test/fixtures/dialects/postgres/postgres_drop_materialized_view.sql

```diff
@@ -0,0 +1,6 @@
+DROP MATERIALIZED VIEW bar;
+DROP MATERIALIZED VIEW bar CASCADE;
+DROP MATERIALIZED VIEW bar RESTRICT;
+DROP MATERIALIZED VIEW IF EXISTS bar;
+DROP MATERIALIZED VIEW IF EXISTS bar CASCADE;
+DROP MATERIALIZED VIEW IF EXISTS bar RESTRICT;
```

Comment:
need unit test for `DROP MATERIALIZED VIEW foo, bar;`

Reply:
Done